src/rdkafka_request.c

```diff
@@ -238,6 +245,55 @@ rd_kafka_buf_read_topic_partitions (rd_kafka_buf_t *rkbuf,
         return NULL;
 }
 
+
+/**
+ * @brief Write a list of topic+partitions to \p rkbuf
+ *
+ * @remark The \p assignment list MUST be sorted.
+ */
+void
+rd_kafka_buf_write_assignment (rd_kafka_buf_t *rkbuf,
```

Comment:
There is a rd_kafka_buf_write_topic_partitions() that can be used

Reply:
very good, missed that.